test/functional/wallet_multiwallet.py

```diff
@@ -68,7 +68,7 @@ def wallet_file(name):
         wallet_names = ['w1', 'w2', 'w3', 'w', 'sub/w5', os.path.join(self.options.tmpdir, 'extern/w6'), 'w7_symlink', 'w8', '']
         extra_args = ['-wallet={}'.format(n) for n in wallet_names]
         self.start_node(0, extra_args)
-        assert_equal(set(map(lambda w: w['name'], self.nodes[0].listwalletdir()['wallets'])), set(['', 'w3', 'w2', 'sub/w5', 'w7', 'w7', 'w1', 'w8', 'w']))
+        assert_equal(sorted(map(lambda w: w['name'], self.nodes[0].listwalletdir()['wallets'])), ['', os.path.join('sub', 'w5'), 'w', 'w1', 'w2', 'w3', 'w7', 'w7_symlink', 'w8'])
```

Comment:
14561 requires 14559 for this to be backported cleanly

Reply:
Interesting, I see 14559 introduced `os.path.join`. Just added 14559 before 14561.  Curious that `wallet_names` doesn't use `os.path.join` and instead hardcodes `/` char.